.github/workflows/release.yaml

```diff
@@ -0,0 +1,42 @@
+name: "Release"
+
+on:
+  workflow_dispatch:
+    inputs:
+      releaseBranch:
+        description: "Release branch name"
+        required: true
+        type: string
+      oldVersion:
+        description: "The last fully released Kuma version"
+        required: true
+        type: string
+      newVersion:
+        description: "New Kuma version"
+        required: true
+        type: string
+
+jobs:
+  create-branch:
+    runs-on: ubuntu-latest
+    env:
+      RELEASE_BRANCH: ${{ github.event.inputs.releaseBranch }}
+      OLD_VERSION: ${{ github.event.inputs.oldVersion }}
+      NEW_VERSION: ${{ github.event.inputs.newVersion }}
+    steps:
+      - name: git checkout
+        uses: actions/checkout@v2
+        with:
+          fetch-depth: 0
+      - name: create branch
+        run: |
+          git --version
+          git config user.name "GitHub Actions Bot"
```

Comment:
```suggestion
          git config user.name "github-actions[bot]"
```

Reply:
I wonder if `gh auth setup-git` would be of any help here.